src/CommunityToolkit.Maui.Core/Handlers/Popup/PopupHandler.macios.cs

```diff
@@ -101,6 +101,6 @@ protected override MauiPopup CreatePlatformElement()
 	protected override void DisconnectHandler(MauiPopup nativeView)
 	{
 		base.DisconnectHandler(nativeView);
-		PlatformView.CleanUp();
+		nativeView.CleanUp();
```

Comment:
can we avoid the usage of `native`? This can be `platformView`

Reply:
I even forgot that native is a forbidden word in .NET MAUI. replaced it everywhere in code, but left in summary